mkdocs.yml

```diff
@@ -113,6 +113,8 @@ nav:
       - geps/gep-1619/index.md
       - geps/gep-1651/index.md
       - geps/gep-1867/index.md
+      - geps/gep-2648/index.md
+      - geps/gep-2649/index.md
     # - Implementable:
     #   -
```

Comment:
Can't comment on the right line for this, but don't forget to move 713 down to memorandum.